### DISCUSSION TITLE
【委托】帮我找找开拓者

### DISCUSSION THREAD

**Discussion by Yy849** [share121/inter-knot#488] [其他]
我们刚到仙舟没多久，开拓者就不见了，不知道去哪了，哪位绳匠能帮我找找看，急！！！

委托奖励：300 菲林   68000 丁尼

**Comment by 1155993355** [comment]
这个委托我接了

**Reply by Frieezed** [reply]
对不起，做不到

**Comment by 1155993355** [comment]
找到了，但是人不回来。

**Comment by ran-mitake-ag** [comment]
我是买1000丁尼萝卜那个，我在空洞里看到个灰毛跑了，应该是那个人吧

**Comment by 1155993355** [comment]
对了，这位帖主，为了完成任务，我决定将用一些强制手段将这位名叫开拓者的人带回家，不知你可否同意？

**Comment by 1155993355** [comment]
可能会受伤，但是如果您不满意的话，价格可以75折。

**Reply by ran-mitake-ag** [reply]
你先别去，我这因为一些原因出不去，治安官也有点迷路找不到我，我可能掉进没有萝卜数据的地方了，三思！

**Comment by Elysia-Ego-093** [comment]
找到了，人在贝洛伯格的垃圾桶里

**Reply by 1155993355** [reply]
怎么样？本绳匠的带路服务如何？

**Comment by yohaumi** [comment]
绳匠们加油，可不能让星核精跑了口牙(╯▔皿▔)╯

**Reply by 1155993355** [reply]
找到你了，抓！
